src/widgetastic/widget.py

```diff
@@ -374,7 +374,11 @@ def root_browser(self):
 
     @property
     def parent_browser(self):
-        return self.parent.browser
+        try:
+            return self.locatable_parent.browser
```

Comment:
Here it is the single condition, so it looks neater. locatable_parent is always present, but if it is none, then it will raise the exception on browser access.